src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -57,6 +57,10 @@ trait FastSync {
     case PeerActor.MessageReceived(BlockHeaders(Seq(blockHeader))) =>
       sender() ! PeerActor.Unsubscribe
 
+      if (appStateStorage.getEstimatedHighestBlock() < blockHeader.number) {
+        appStateStorage.putEstimatedHighestBlock(blockHeader.number)
+      }
```

Comment:
It can be, but it wouldn't be particularly accurate estimate of highest block on the blockchain :)